examples/structured_data/md/collaborative_filtering_movielens.md

```diff
@@ -166,20 +176,26 @@ class RecommenderNet(keras.Model):
         user_bias = self.user_bias(inputs[:, 0])
         movie_vector = self.movie_embedding(inputs[:, 1])
         movie_bias = self.movie_bias(inputs[:, 1])
-        dot_user_movie = tf.tensordot(user_vector, movie_vector, 2)
+        dot_user_movie = ops.tensordot(user_vector, movie_vector, 2)
         # Add all the components (including bias)
         x = dot_user_movie + user_bias + movie_bias
         # The sigmoid activation forces the rating to between 0 and 1
-        return tf.nn.sigmoid(x)
+        return ops.nn.sigmoid(x)
 
 
 model = RecommenderNet(num_users, num_movies, EMBEDDING_SIZE)
 model.compile(
-    loss=tf.keras.losses.BinaryCrossentropy(),
+    loss=keras.losses.BinaryCrossentropy(),
     optimizer=keras.optimizers.Adam(learning_rate=0.001),
 )
 ```
 
+<div class="k-default-codeblock">
+```
+An NVIDIA GPU may be present on this machine, but a CUDA-enabled jaxlib is not installed. Falling back to cpu.
```

Comment:
This is a big problem! You may want to fix that.

Reply:
Reinstalled conda so likely have to redo some gpu things, thanks for finding! (will remove)

Reply:
Fixed!